src/shared/proto/tbots_robot_msg.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto2";
+
+import "shared/proto/tbots_robot_status_2014.proto";
+import "shared/proto/tbots_frankie_status.proto";
+
+message Tbots_RobotMsg {
+
+    // The robot id.
+    required int32 robot_id = 4;
+
+    // Messages related to the robot that are to be displayed in the visualizer
+    repeated string robot_messages = 5;
+
+    // Messages that are related to network communication
+    repeated string network_messages = 6;
```

Comment:
ditto previous comment, is this just a network log from the robot? What would we use the `repeated string` `network_messages` for over `robot_messages`?